src/thunderbots/software/CMakeLists.txt

```diff
@@ -257,6 +257,7 @@ if (CATKIN_ENABLE_TESTING)
             ai/primitive/kick_primitive.cpp
             ai/primitive/move_primitive.cpp
             ai/primitive/primitive.cpp
+	    ai/primitive/pivot_primitive.cpp
```

Comment:
Just taking a guess here, this might be using tabs instead of spaces. I'd recommend making sure whatever editor you're using inserts spaces instead of tabs when you hit the "Tab" key.

Reply:
I think I got them all